### DISCUSSION TITLE
Tamanho dos inputs grandes

### DISCUSSION THREAD

**Discussion by dabraga** [po-ui/po-angular#1322] [Tire suas dúvidas] [answered]
Boa tarde , 
Estou fazendo uma pagina e notei que os input ficam grande , queria diminuir.. Existe essa possibilidade ?

**Comment by anliben** [comment, marked as answer]
Bom Dia @dabraga, tudo bem?

Confira o nosso construtor de temas, nele consegue personalizar componentes e copiar o CSS inclusive!

Attr.
